wasm/jit/jit_amd64.go

```diff
@@ -1660,3 +1763,18 @@ func (b *amd64Builder) initializeReservedRegisters() {
 	prog.From.Reg = reg
 	b.addInstruction(prog)
 }
+
+// ensureLocatedOnGeneralPurposeRegister ensures that the given value is located on a
+// general purpose register of an appropriate type.
+func (b *amd64Builder) ensureLocatedOnGeneralPurposeRegister(loc *valueLocation) error {
```

Comment:
since the parameter is a location, can we remove located from this function name?

Reply:
https://github.com/tetratelabs/wazero/pull/89/commits/cec121be1b0bf49a5f1cf57d395cb30f67520622